dzVents/runtime/device-adapters/opentherm_gateway_device.lua

```diff
@@ -19,7 +19,7 @@ return {
 		function device.updateSetPoint(setPoint)
 			-- send the command using openURL otherwise, due to a bug in Domoticz, you will get a timeout on the script
 			local url = domoticz.settings['Domoticz url'] ..
-					'/json.htm?param=udevice&type=command&idx=' .. device.id .. '&nvalue=0&svalue=' .. setPoint
+					'/json.htm?param=udevice&type=command&idx=' .. device.id .. '&nvalue=' .. toNumber(setPoint) .. '&svalue=' .. setPoint
```

Comment:
A setpoint device stores its temperature in svalue (like 21.5) , not in the nvalue (which is a integer)